Goobi/src/de/sub/goobi/metadaten/copier/DataCopier.java

```diff
@@ -82,7 +85,14 @@ public DataCopier(String program) throws ConfigurationException {
 	 */
 	public void process(CopierData data) {
 		for (DataCopyrule rule : rules) {
-			rule.apply(data);
+			try {
+				rule.apply(data);
+			} catch (RuntimeException nA) {
+				if (LOG.isInfoEnabled()) {
+					LOG.info("Metadata copy rule »" + rule + "« not applicable for process »" + data.getProcessTitle()
+							+ "«. Rule skipped.");
+				}
```

Comment:
I use it because the rules contain lots of quotes themselves to make the log mor readable